.circleci/config.yml

```diff
@@ -146,6 +169,9 @@ jobs:
       - run:
           name: Run tests
           command: |
+             if [ << parameters.OQS_PROVIDER_BUILD_STATIC >> -eq "true" ]; then
+               export OQS_PROVIDER_BUILD_STATIC=1
```

Comment:
What about only starting `ctest` in this case and leave `runtests.sh` untouched (and run in the "else" branch)? The changes to `runtests.sh` would be pretty significant: It contains _lots of_ `openssl` invocations that would need to be guarded.